packages/react/src/components/media/controls/AirPlay.tsx

```diff
@@ -0,0 +1,84 @@
+import { MediaControllerState } from 'livepeer';
+import { isAirPlaySupported } from 'livepeer/media/browser';
+import { styling } from 'livepeer/media/browser/styling';
+import * as React from 'react';
+
+import { useMediaController } from '../../../context';
+import { PropsOf } from '../../system';
+const DefaultAirPlayIcon = () => (
+  <svg
+    width="100%"
+    height="100%"
+    viewBox="0 0 36 36"
+    fill="none"
+    style={{
```

Comment:
Do we need these styles?

Reply:
I believe those styling the necessary to make sure the icon is aligned with PiP and fullscreen icon.